src/assets/i18n/pt-BR.json5

```diff
@@ -8345,96 +7744,73 @@
+  "person.page.orcid.synchronization-mode-funding-message": "Selecione se deseja enviar suas entidades de projeto vinculadas à lista de informações de financiamento do seu registro ORCID.",
 
   // "person.page.orcid.synchronization-mode-publication-message": "Select whether to send your linked Publication entities to your ORCID record's list of works.",
-  // TODO New key - Add a translation
-  "person.page.orcid.synchronization-mode-publication-message": "Select whether to send your linked Publication entities to your ORCID record's list of works.",
+  "person.page.orcid.synchronization-mode-publication-message": "Selecione se deseja enviar suas entidades de publicação vinculadas à lista de trabalhos do seu registro ORCID.",
 
   // "person.page.orcid.synchronization-mode-profile-message": "Select whether to send your biographical data or personal identifiers to your ORCID record.",
-  // TODO New key - Add a translation
-  "person.page.orcid.synchronization-mode-profile-message": "Select whether to send your biographical data or personal identifiers to your ORCID record.",
+  "person.page.orcid.synchronization-mode-profile-message": "Selecione se deseja enviar seus dados biográficos ou identificadores pessoais para seu registro ORCID.",
 
   // "person.page.orcid.synchronization-settings-update.success": "The synchronization settings have been updated successfully",
-  // TODO New key - Add a translation
-  "person.page.orcid.synchronization-settings-update.success": "The synchronization settings have been updated successfully",
+  "person.page.orcid.synchronization-settings-update.success": "As configurações de sincronização foram atualizadas com sucesso",
 
   // "person.page.orcid.synchronization-settings-update.error": "The update of the synchronization settings failed",
-  // TODO New key - Add a translation
-  "person.page.orcid.synchronization-settings-update.error": "The update of the synchronization settings failed",
+  "person.page.orcid.synchronization-settings-update.error": "A atualização das configurações de sincronização falhou",
 
   // "person.page.orcid.synchronization-mode.manual": "Manual",
-  // TODO New key - Add a translation
   "person.page.orcid.synchronization-mode.manual": "Manual",
 
   // "person.page.orcid.scope.authenticate": "Get your ORCID iD",
-  // TODO New key - Add a translation
-  "person.page.orcid.scope.authenticate": "Get your ORCID iD",
+  "person.page.orcid.scope.authenticate": "COnseguir seu ORCID iD",
 
   // "person.page.orcid.scope.read-limited": "Read your information with visibility set to Trusted Parties",
-  // TODO New key - Add a translation
-  "person.page.orcid.scope.read-limited": "Read your information with visibility set to Trusted Parties",
+  "person.page.orcid.scope.read-limited": "Leia suas informações com visibilidade definida como Parceiros Confiáveis",
```

Comment:
`Aceda às suas informações com a visibilidade definida para Parceiros Confiáveis`

Reply:
Aceda is not common in  Brazilian Portuguese